hnn_core/batch_simulate.py

```diff
@@ -252,15 +299,39 @@ def _run_single_sim(self, param_values):
             net = calcium_model()
 
         self.set_params(param_values, net)
```

Comment:
We should consider how to integrate #763 with this PR such that users can load in base networks of their own choosing

Reply:
Read some of the code in #837 to see how instantiating networks from the `json` format looks

Reply:
Here is the function that converts exported `json` files to a network https://github.com/jonescompneurolab/hnn-core/blob/773a86d9955eadb6ab4eae39bd9c3f2e91c8f403/hnn_core/hnn_io.py#L384